### PR TITLE
Avoid loading ActiveRecord::Base early

### PR DESCRIPTION
**Pull request by casperisfine**
Doing this cause a slew of problems with Rails loading process, causing some configurations not to be applied, etc.

Fix: #89
Fix: #90

### CODE REVIEW

**Comment by benoittgt** [comment]
I confirm that is a good proposal that fix some issue on other gem.

**Review by etagwerker — APPROVED** [review]
@casperisfine Thanks for submitting this patch! 

Just waiting for CI to finish before I merge.

**Pull request merged**
